modules/react/modal/lib/ModalOverlay.tsx

```diff
@@ -48,14 +58,14 @@ const Container = styled(Box)<StyledType>({
 const ResponsiveContainer = styled('div')(({theme}) => {
   const {canvas: canvasTheme} = getTheme(theme);
   return {
-    maxHeight: '100vh',
+    maxHeight: '100%',
     display: 'flex',
     position: 'absolute',
     left: 0,
     top: 0,
     justifyContent: 'center',
     alignItems: 'center',
-    height: '100vh',
+    height: '100%',
```

Comment:
Not sure if this might break other applications

Reply:
The time it would be a problem is if the parent container of the overlay isn't 100% of the height of the viewable area since `100%` sets the height of that element to `100%` of the height of it's parent element.